WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -61,13 +61,10 @@ extension UIImageView {
     ///
     /// - Parameters:
     ///     - request: the request for the SiteIcon.
-    ///     - placeholderImage: Yes. It's the "place holder image".
+    ///     - config: Includes parameters to take into consideration when downloading the blog image.
     ///
-    private func downloadSiteIcon(
-        with request: URLRequest,
-        placeholderImage: UIImage?) {
-
-        af_setImage(withURLRequest: request, placeholderImage: placeholderImage, completion: { [weak self] dataResponse in
+    private func downloadSiteIcon(with request: URLRequest, config: SiteIconDownloadConfig) {
```

Comment:
I am not opposed to add another method, but how about we use just one and add an `imageSize` parameter which defaults to the default value of 40? This way we don't need another dedicated type and it feels a bit lighter, wdyt?